CHANGES.md

```diff
@@ -1,6 +1,437 @@
+* when multiple supported LLVM versions are found, the latest one is now chosen
+* updated error message when unable to find LLVM to list the supported versions
+* UTF-8 support no longer depends on the `LANG` environment variable  
+  (see https://chapel-lang.org/docs/1.26/usingchapel/chplenv.html#character-set)
+* removed support for the deprecated `CHPL_REGEXP` environment variable
+
+Syntactic / Naming Changes
+--------------------------
+* extended the set of statement types to which `try`/`try!` can be applied
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* top-level modules defined in files on `chpl`'s command-line are now inited  
+  (see https://chapel-lang.org/docs/1.26/language/spec/modules.html#module-initialization)
+* enabled implicit conversions from non-nilable to nilable for type arguments  
+  (see https://chapel-lang.org/docs/1.26/language/spec/procedures.html#legal-argument-mapping)
+* `.indices` queries on rectangular arrays now return a local domain of indices  
+  (see https://chapel-lang.org/docs/1.26/language/spec/arrays.html#ChapelArray.indices)
+* `.size`/`.shape` queries on ranges, domains, arrays now return `int`s  
+  (see https://chapel-lang.org/docs/1.26/builtins/ChapelRange.html#ChapelRange.range.size)
+* stopped auto-creating `hash()` methods for records w/ custom `==`/`!=` ops  
+  (see https://chapel-lang.org/docs/1.26/language/spec/records.html#hashing-a-record)
+* made it an error to overload return intents for the `enterThis()` method  
+  (see https://chapel-lang.org/docs/1.26/language/spec/statements.html#the-manage-statement)
+
+New Features
+------------
+* added support for listing operators in `only`, `except`, and `import`  
+  (see https://chapel-lang.org/docs/1.26/language/spec/modules.html#using-modules  
+   and https://chapel-lang.org/docs/1.26/language/spec/modules.html#importing-modules)
+* added an `unsafeAssign()` method and context manager to domains  
+  (see https://chapel-lang.org/docs/1.26/language/spec/domains.html#ChapelDomain.unsafeAssignManager)
+* added support for renaming `extern type` declarations  
+  (e.g., `extern "c_name" type chpl_name = ...;`  
+   see also https://chapel-lang.org/docs/1.26/language/spec/interoperability.html#referring-to-external-c-types)
+
+Feature Improvements
+--------------------
+* extended hash-based collections to support values of range/array type
+* improved support for core operations across rectangular array implementations
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* deprecated support for `|`, `&`, and `^` on rectangular domains
+* deprecated `domain.isSuper()` and `.isSubset()` for `domain.contains()`
+* deprecated support for arrays over brace-less anonymous associative domains  
+  (e.g., in `var A: ["red", "green", "blue"] real;`, brackets are now required)
```

Comment:
Could/should it say 'braces' or 'curly braces' or '{} brackets" instead of just 'brackets' ?